packages/esm-service-queues-app/src/send-patient-back-toqueue/send-back-patient-toqueue.component.tsx

```diff
@@ -0,0 +1,286 @@
+            <Controller
+              name="location"
+              control={control}
+              defaultValue={data?.queueComingFrom?.location?.uuid}
+              render={({ field: { onChange, value } }) => (
+                <Select
+                  labelText={t('selectQueueLocation', 'Select a queue location')}
+                  id="location"
+                  invalid={!!errors.location}
+                  invalidText={errors.location?.message}
+                  value={value}
+                  onChange={(event) => onChange(event.target.value)}>
+                  {!getValues()?.location && (
+                    <SelectItem text={t('selectQueueLocation', 'Select a queue location')} value="" />
+                  )}
+                  {queueLocations?.length > 0 &&
+                    queueLocations.map((location) => (
+                      <SelectItem key={location.id} text={location.name} value={location.id}>
+                        {location.name}
+                      </SelectItem>
+                    ))}
+                </Select>
+              )}
+            />
+          </section>
+
+          <section className={styles.section}>
+            <div className={styles.sectionTitle}>{t('queueService', 'Queue service')}</div>
+            <Controller
+              name="service"
+              defaultValue={data?.queueComingFrom?.uuid}
+              control={control}
+              render={({ field: { onChange, value } }) => (
+                <Select
+                  labelText={t('selectService', 'Select a service')}
+                  id="service"
+                  invalid={!!errors.service}
+                  invalidText={errors.service?.message}
+                  value={value}
+                  onChange={(event) => onChange(event.target.value)}>
+                  {!getValues()?.service && <SelectItem text={t('selectService', 'Select a service')} value="" />}
+                  {queues?.length > 0 &&
+                    queues.map((service) => (
+                      <SelectItem key={service.uuid} text={service.display} value={service.uuid}>
+                        {service.display}
+                      </SelectItem>
+                    ))}
+                </Select>
+              )}
+            />
+          </section>
+
+          <section className={styles.section}>
+            <div className={styles.sectionTitle}>{t('queueStatus', 'Queue status')}</div>
+            {!allowedStatuses?.length ? (
+              <InlineNotification
+                className={styles.inlineNotification}
+                kind={'error'}
+                lowContrast
+                subtitle={t('configureStatus', 'Please configure status to continue.')}
+                title={t('noStatusConfigured', 'No status configured')}
+              />
+            ) : (
+              <Controller
+                name="status"
+                control={control}
+                defaultValue={data?.queue?.allowedStatuses[2].uuid}
```

Comment:
will allowedStatuses[2] always be what you want? It is possible to be more expressive?